network/load-balancer/how-to/create-load-balancer.mdx

```diff
@@ -21,37 +21,45 @@ dates:
 1. Click **Load Balancers** in the **Network** section of the [Scaleway Console](https://console.scaleway.com) side menu. If you have not already created Load Balancer, the product creation page is displayed:
     <Lightbox src="scaleway-lb-creation-page.webp" />
 2. Click **Create a Load Balancer**. The creation wizard displays.
-3. Complete the following steps in the wizard: 
-    - Enter the Load Balancer **name** and **description**. Optionally, you can assign **tags** to help organize your Load Balancers.
-    - Choose the **Availability Zone** in which your Load Balancer will be deployed geographically. Currently we provide the following Availability Zones:
+
+3. Enter a **name** and **description** for your Load Balancer. Optionally, you can assign **tags** to help organize your Load Balancers.
```

Comment:
The description is optional, do you think we should mention it in the second sentence instead?

Reply:
yes, done